api4/job.go

```diff
@@ -35,6 +37,37 @@ func getJob(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	forceDownload, _ := strconv.ParseBool(r.URL.Query().Get("download"))
+	if forceDownload {
+		fileName := "actiance_export.xml"
+		fileMime := "application/xml"
+		if job.Data["export_type"] == "csv" {
+			fileName = "csv_export.zip"
+			fileMime = "applicaiton/zip"
+		}
+
+		filePath := "export/" + job.Id + "/" + fileName
+		data, readFileErr := c.App.ReadFile(filePath)
+		if readFileErr != nil {
+			c.Err = readFileErr
```

Comment:
Good point. I just did a call on a valid job ID which I deleted the file for and called the endpoint, I got the following returned. It is returning the file path and it includes file name. Why are you suggesting that we don't want to return the file name?
```
{
    "id": "api.file.read_file.reading_local.app_error",
    "message": "Encountered an error reading from local server file storage.",
    "detailed_error": "open data/export/ztged6659i85fjkzqzb7o7fyuh/actiance_export.xml: no such file or directory",
    "request_id": "b7rog6dqrir95japc65kth4z7a",
    "status_code": 500
}
```

Reply:
Check up with security on returning the full file path or not (currently happening for file uploads on chat facing side) 
We ideally shouldn't be returning the full path.

Reply:
I did notice that for the file uploads, the path was exposed if I manually deleted the file from the directory or the directory itself. We don't have an endpoint to delete the file directly. I was testing in community-daily where I uploaded the file and deleted the post and then tried doing a get call on that file ID. It returned an error saying that the file ID was not found in the database (indicating deletion of the file information in the database). 

However, regardless, I have sanitized the return error to blank out the detailed_mesage just in case as the message seems to be enough information to indicate the problem.

Reply:
I don't see an issue here. The only thing revealed is the path to the data directory (which is just `data`) buy default, and the rest is either random or static.

Reply:
Information disclosures like this would be a bigger issue if we were closed-source, but as it stands everything can be read from the sources anyway.

Reply:
@jupenur Thank you Juhno for the input!